### ISSUE TITLE
Panic when handling Subscribe RPC call.

### ISSUE THREAD

**Issue by lthibault** [wetware/ww#18]
Encountered the following panic when developing an application against single-node ww instance, running locally.
```
$ ww start
INFO[0000] wetware loaded                                id=QmbhXNu1qyHKmWSdaSGGXMPXd7WZVa1GVbTMURACQXws1V instance=0e265378-c2ca-477c-afdf-e73997fb7c5f ns=bn version=0.0.0
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x440c7e6]

goroutine 296 [running]:
capnproto.org/go/capnp/v3/rpc.(*importClient).Send.func4()
        /Users/lthibault/Go/pkg/mod/capnproto.org/go/capnp/v3@v3.0.0-alpha.2/rpc/import.go:127 +0x46
github.com/wetware/ww/pkg/cap/pubsub.(*refCountedTopic).send(0xc001fa9940, {0x52b0160?}, 0xc00033a600?)
        /Users/lthibault/Go/src/github.com/wetware/ww/pkg/cap/pubsub/server.go:243 +0xaa
github.com/wetware/ww/pkg/cap/pubsub.(*refCountedTopic).handle(0xc001fa9940, 0xc00228ac80, {0xc001fa9980?})
        /Users/lthibault/Go/src/github.com/wetware/ww/pkg/cap/pubsub/server.go:232 +0xee
created by github.com/wetware/ww/pkg/cap/pubsub.(*refCountedTopic).Subscribe
        /Users/lthibault/Go/src/github.com/wetware/ww/pkg/cap/pubsub/server.go:216 +0x20a
```

Triggering event is unclear.  Appears to have happened when the process publishing to topic was terminated concurrently to `Subscribe` call above.

**Comment by lthibault** [comment]
Possibly resolved by moving to capnp v3.0.0-alpha.3.  Keeping this open for now.

**Comment by lthibault** [comment]
Haven't seen this in a while.  I now suspect something involving a capnp version mismatch.  Closing optimistically.
